src/aws/osml/model_runner/region_request_handler.py

```diff
@@ -0,0 +1,205 @@
+#  Copyright 2023-2024 Amazon.com, Inc. or its affiliates.
+
+import logging
+from typing import Optional
+
+from aws_embedded_metrics.logger.metrics_logger import MetricsLogger
+from aws_embedded_metrics.metric_scope import metric_scope
+from aws_embedded_metrics.unit import Unit
+from osgeo import gdal
+
+from aws.osml.photogrammetry import SensorModel
+
+from .api import RegionRequest
+from .app_config import MetricLabels, ServiceConfig
+from .common import EndpointUtils, RequestStatus, Timer
+from .database import EndpointStatisticsTable, JobItem, JobTable, RegionRequestItem, RegionRequestTable
+from .exceptions import ProcessRegionException, SelfThrottledRegionException
+from .queue import RequestQueue
+from .status import RegionStatusMonitor
+from .tile_worker import TilingStrategy, process_tiles, setup_tile_workers
+
+# Set up logging configuration
+logger = logging.getLogger(__name__)
+
+# GDAL 4.0 will begin using exceptions as the default; at this point the software is written to assume
+# no exceptions so we call this explicitly until the software can be updated to match.
+gdal.UseExceptions()
```

Comment:
This seems out of place here.  If we only need to call this once I feel like it should be with app.py still, maybe in init.